graphql_compiler/tests/test_input_data.py

```diff
@@ -1594,21 +1594,21 @@ def traverse_and_optional_and_traverse():
         Animal {
             name @output(out_name: "animal_name")
             in_Animal_ParentOf {
-                name @output(out_name: "parent_name")
+                name @output(out_name: "child_name")
                 out_Animal_ParentOf @optional {
-                    name @output(out_name: "sibling_name")
+                    name @output(out_name: "spouse_name")
```

Comment:
make this `spouse_and_self` for consistency with `spouse_and_self_species` below -- the "self" is definitely included